jcapi-systems.go

```diff
@@ -71,9 +71,20 @@ func (jcsystem JCSystem) ToString() string {
 	for _, tag := range jcsystem.Tags {
 		returnVal += fmt.Sprintf("\t%s\n", tag.ToString())
 	}
+
 	return returnVal
 }
 
+func (jcsystem JCSystem) SystemHasTag(tagName string) (bool, string) {
```

Comment:
I've really taken to specifying my return variable names in the returns list, it has a few benefits:

1. It can make the code more readable.
2. You don't have to separately declare your variables.
3. Regardless of where you call 'return' from, you always get the latest settings for those variables.

Just overall much cleaner IMO.

In this case, I would do:

func (jcsystem JCSystem) SystemHasTag(tagName string) (hasTag bool, tagId string) {